src/components/List/ListItem.tsx

```diff
@@ -37,9 +39,11 @@ function ListItem<T>({
   const [expanded, setExpanded] = useState(defaultExpanded);
   const [id] = useState(() => uniqueId('listitem-'));
 
+  const unselectedColor = expanded ? expandedColor : color;
```

Comment:
Do you think we should use `color` as a fallback in case `expandedColor` is undefined? It is a new prop and will change the behavior if not passed in.
```ts
const unselectedColor = expanded && expandedColor || color;
```

Reply:
Well, luckily in this case, color is always undefined!  But we can certainly be more rigorous about this prop.